docs/12rpcs/nft_rpcs.mdx

```diff
@@ -315,7 +315,7 @@ Request Parameters:
 | :------------- | :------- | :------------------------------------------------------------------------------------------------------------------ |
 | wallet_id      | True     | The Wallet ID of the NFT to transfer                                                                                |
 | target_address | True     | The address to transfer the NFT to. For NFT0 this must be an XCH address. For NFT1 this could also be a DID address |
-| nft_coin_id    | True     | The coin ID of the NFT to transfer                                                                                  |
+| nft_coin_id    | True     | The coin ID of the NFT to transfer. This can be obtained by running the [nft_get_nfts](#nft_get_nfts) RPC and selecting the value of `nft_coin_id` from the output |
```

Comment:
I think it would be better to put this into an admonition, either below each command's table that references it, or on a single place on the page with a link to it from each relevant parameter description